bin/download_dac.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+# coding=utf-8
+
+"""
+Script that scrapes CONTACT_NAME from Departmental Audit Committee
+Members Appointed and Non-Appointed dataset. The dataset is available on the
+Open Government Portal:
+https://open.canada.ca/data/en/dataset/1f681b53-bba4-437b-96b0-78af262e0ff8
+
+USAGE: python download_dac.py -o/--output outfile.yaml
+
+"""
+
+import codecs
+import getopt
+import json
+import sys
+from urllib import urlopen
+
+
+def main(argv):
+    try:
+        outfile = ''
+        opts, args = getopt.getopt(argv, "o:", ["output="])
+        for opt, arg in opts:
+            if opt in ("-o", "--output"):
+                outfile = arg
+        if not outfile:
+            raise ValueError
+
+    except ValueError:
+        print 'USAGE: python download_dac.py -o/--output outfile.yaml'
+        sys.exit(1)
+    except getopt.GetoptError:
+        print 'USAGE: python download_dac.py -o/--output outfile.yaml'
+        sys.exit(1)
```

Comment:
this should work too
```suggestion
    except (ValueError, getopt.GetoptError):
        print 'USAGE: python download_dac.py -o/--output outfile.yaml'
        sys.exit(1)
```